src/main/resources/assets/ts/example.ts

```diff
@@ -0,0 +1,7 @@
+const example = () => {
+  const msg: string = 'Hello from transpiled Typescript client-side code.';
+  console.log(msg);
+};
+
+export default example;
+
```

Comment:
even if we are going to use typescript for this project these file and the next one are examples we are not going to use. Storybook will serve the necessary javascript for the frontend in Enonic's folder anyway. I suggest to remove both files to not confuse